### ISSUE TITLE
Remove Rate and Index values from Labels

### ISSUE THREAD

**Issue by rerobins** [jenningsloy318/panos_exporter#7]
Per the prometheus documentation:

> CAUTION: Remember that every unique combination of key-value label pairs represents a new time series, which can dramatically increase the amount of data stored. Do not use labels to store dimensions with high cardinality (many different label values), such as user IDs, email addresses, or other unbounded sets of values.

This means that labels such as `rate` and `index` are causing new timeseries values to be created and prevents comparisons as unless the rate and the index are the same.  I recommend removing those fields from the list of labels that are being fetched.  Rate is redundant as Prometheus provides that as a function.  I do not know what value index provides.

In the mean time, the following configuration (see `metric_relabel_configs`) removes those labels.

```yaml
  - job_name: panos_exporter
    metrics_path: /panos
    static_configs:
      - targets:
        - target01
        - target02
        - target03
    relabel_configs:
      - source_labels: [__address__]
        target_label: __param_target
      - source_labels: [__param_target]
        target_label: instance
      - target_label: __address__
        replacement: panos_exporter:9654
    metric_relabel_configs:
      - regex: 'rate'
        action: labeldrop
      - regex: 'id'
        action: labeldrop 
```

**Comment by jenningsloy318** [comment]
Thanks for you info, actually these labels are auto-generated thus I didn't deliberately added them in the code. 

Nevertheless you are free to add these config in your job definitions.

I will also add it to the document

**Comment by pawelos99** [comment]
Fix added to PR, with that fix dropping labels in prometheus config won't be needed anymore.

**Comment by zbbfufu** [comment]
Fixed by #21 and #22
